File: getMethodSignatures.java
import jeb.api.IScript;
/*import jeb.api.ui.View;*/
import jeb.api.dex.Dex;
import jeb.api.JebInstance;
import java.util.*;
import java.io.*;

public class getMethodSignatures implements IScript {
    
    private static String path = "/tmp/methodsignatures.txt";

    public void run(JebInstance jeb){
        Dex dex  = jeb.getDex();
        List<String> methodSigList = new ArrayList<String>();

        methodSigList = dex.getMethodSignatures(true);

        try{
        	File file = new File(path);
	        FileOutputStream fos =  new FileOutputStream(file);
	        
	        for (String method : methodSigList)
	        	fos.write((method + "\n").getBytes());	            
        }catch(Exception e){
        	
        }

        jeb.print("[+] Output file located at "+path);
        jeb.print("[+] Done!");
    }

}

File: getMethodSignatures.py
# -*- coding: utf-8 -*-

import codecs # used for writing UTF-8 into a file

from jeb.api     import IScript
from jeb.api.ui  import View
from jeb.api.dex import Dex
from jeb.api.ast import Class, Field, Method, Call, Constant, StaticField, NewArray, Compound, Assignment, Identifier, Optimizer


class getMethodSignatures(IScript):

  def run(self, jeb):
    self.jeb = jeb
    self.dex = self.jeb.getDex()
    self.path = "/tmp/methodsignatures.txt"

    methodSigList = self.dex.getMethodSignatures(True)

    #f = codecs.open(self.path, "w", encoding="ascii", errors="ignore")
    f = codecs.open(self.path, "w", encoding="utf-8", errors="ignore")

    for method in methodSigList:
        f.write(repr(method)+'\n')
	     	
    print "[+] Output file located at %s" %(self.path)
    print "[+] Done!"
